groovy/postInit/chemistry/organic_chemistry/OilOverhaul.groovy

```diff
@@ -1277,6 +1277,14 @@ ALLOY_SMELTER.recipeBuilder()
         .EUt(30)
         .buildAndRegister()
 
+ALLOY_SMELTER.recipeBuilder()
```

Comment:
what is the purpose of this, it is literally an exact duplicate

Reply:
ingotLead instead of dustLead